py3.10-google-cloud-sdk.yaml

```diff
@@ -47,30 +55,39 @@ pipeline:
       rm -rf google-cloud-sdk/deb
       rm -rf google-cloud-sdk/rpm
       rm google-cloud-sdk/install.bat
-      rm google-cloud-sdk/data/cli/gcloud.json
 
       # Remove any third party tests that bloat up the package
       find google-cloud-sdk -type d \( -name 'tests' -o -name 'test' \) -path '*/third_party/*' -exec rm -rf {} +
 
-      # This is a large binary with vulnerabilities in it, users can add it back later.
-      google-cloud-sdk/bin/gcloud components remove anthoscli
-
       # This is a large binary with vulnerabilities in it, users can add it back later.
       google-cloud-sdk/bin/gcloud components remove gcloud-crc32c
 
       # Running gcloud adds a bunch of pyc files, remove those
       find google-cloud-sdk/ -name "*.pyc" -exec rm -rf '{}' +
       rm -rf google-cloud-sdk/.install
 
+      # gcloud expects to find a directory called ".install" in its "Installation Root" (as reported by "gcloud info").
+      # Without this, "gcloud components" doesn't work.
+      mkdir google-cloud-sdk/.install
+
       mv google-cloud-sdk ${{targets.destdir}}/usr/share/
 
       mkdir -p ${{targets.destdir}}/usr/bin
-      for f in anthoscli bq docker-credential-gcloud gcloud gcloud-crc32c git-credential-gcloud.sh gsutil java_dev_appserver.sh; do
+      for f in bq docker-credential-gcloud gcloud gcloud-crc32c git-credential-gcloud.sh gsutil java_dev_appserver.sh; do
         ln -sf /usr/share/google-cloud-sdk/bin/$f ${{targets.destdir}}/usr/bin/$f
       done
 
   - uses: strip
 
+test:
+  pipeline:
+    - runs: gcloud --version
+    - runs: gsutil --version
+    - runs: gcloud components list
+
+# I don't think this update works as the last tag is 427.0.0 here while current is 487.0.0,
```

Comment:
We don't unfortunately have a way today to automate updating multiple SHAs for packages with multiple checkouts / fetches.

Though having this enabled will still cause a PR to be created whenever an update is required, that'll require manual intervention - so likely why we enable here